### PR TITLE
Kiran-512

### PR DESCRIPTION
**Pull request by Kiran-512**
Hi,

I have completed the portfolio with html and CSS. Please Review the JS code, it's creating trouble for me.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
+/*==================== MENU SHOW Y HIDDEN ====================*/
```
Comment (faran4engg) [inline assets/js/main.js:1]: it looks like you haven't included it in HTML file :)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,998 @@
...
+          </div>
+        </div>
+      </section>
+
+      <!--==================== ABOUT ====================-->
+      <section class="about section" id="about">
+        <h2 class="section__title">About me</h2>
+        <span class="section__subtitle">My Introduction</span>
+        <div class="about__container container grid">
+          <img src="assets/img/about.jpg" alt="" class="about__img" />
```
Comment (Abbas4tech) [inline index.html:162]: alt ? @Kiran-512

File: index.html [hunk 3]
```diff
@@ -0,0 +1,998 @@
...
+                <span class="about__info-title">20+</span>
+                <span class="about__info-name"
+                  >Completed <br />
+                  Projects</span
+                >
+              </div>
+              <div>
+                <span class="about__info-title">05+</span>
+                <span class="about__info-name"
+                  >Companies <br />
```
Comment (Abbas4tech) [inline index.html:187]: `<br/>` is this working?

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
You had to create a new branch for every project @Kiran-512 !!!

File: index.html (hunk 2 above)
Comment (Kiran-512) [reply index.html:162]: Noted and added alt now

File: index.html (hunk 3 above)
Comment (Kiran-512) [reply index.html:187]: Yes, it worked.

**Pull request closed without merging**